pkg/pods/pod.go

```diff
@@ -713,28 +733,34 @@ func (pod *Pod) getLaunchable(launchableID launch.LaunchableID, launchableStanza
 		pod.logger.WithError(err).Warnf("Could not parse version from launchable %s.", launchableID)
 	}
 
+	podCgroup := ""
+	if *NestedCgroups {
+		err = pod.installCgroup()
+		if err != nil {
+			pod.logger.WithError(err).Errorf("Could not create pod cgroup")
```

Comment:
what's the error scenario here? will the process be able to start successfully?

Reply:
Oh good point. I thought WithError actually killed it, but looks like it doesn't.

I think, in that case, it should attempt to try with just the launchable, basically flip the *NestedCgroups flag and try again?

Reply:
I think I will write a cleanup function to os.Remove everything that was created, if it was created